tests/CollectionTest.php

```diff
@@ -10,7 +10,7 @@
 class CollectionTest extends TestCase
 {
     /** @test */
-    public function test_collection()
```

Comment:
Not sure if you prefer to use both `@test` annotation and prefix - it's a bit redundant, so I removed the prefix.

Reply:
Feel free to suggest better names. I have the habit to use a generic name for a test if I can't come up with a better one in a few seconds, thinking I can always change it later.